src/services/control-plane/adapters/drivers/control-authenticating-proxy.ts

```diff
@@ -0,0 +1,16 @@
+import { ControlPlane } from "../../app/control-plane";
+import { AuthDetails, Permissions } from "../../app/shemas";
+import { ForAuthenticatingUser } from "../../ports/drivers";
+
+export class ControlAuthenticatingProxyAdapter
+  implements ForAuthenticatingUser
+{
+  constructor(private readonly controlPlane: ControlPlane) {}
+
+  async getAuthDetails(email: string, password: string): Promise<AuthDetails> {
+    return this.controlPlane.getAuthDetails(email, password);
+  }
```

Comment:
Agregar espacio entre métodos y secciones lógicas
```suggestion
  }
  
```